parser_test.go

```diff
@@ -70,3 +70,49 @@ func TestDrawLabelAsPng(t *testing.T) {
 		})
 	}
 }
+
+func TestDrawLabelAsPng_Barcode128(t *testing.T) {
```

Comment:
Can be merged with the main test ( TestDrawLabelAsPng)

Reply:
yes it can, merged